src/sass/app/paymentagent_transfer.scss

```diff
@@ -0,0 +1,4 @@
+.paymentagent_description {
+    height: 4em;
+    width: 206px;
+}
```

Comment:
if it's a small amount of css, we generally put it in `pages.scss` for static and `websocket_pages.scss` for websocket pages